api/base/serializers.py

```diff
@@ -292,6 +292,8 @@ def __init__(self, source=None, **kwargs):
     def get_attribute(self, obj):
         user = self.context['request'].user
         auth = auth_core.Auth(user)
+        if 'view_only' in self.context['request'].query_params:
+            auth.private_key = self.context['request'].query_params['view_only']
```

Comment:
What's this for? It doesn't seem to be used and I don't _think_ it should be necessary.